src/api-server.ts

```diff
@@ -480,7 +496,7 @@ async function publishToRoom(roomName: string, message: string): Promise<any> {
     }
     else {
         const room = subscriptions.get(roomName);
-        await room?.broadcast(message);
+        await room?.broadcast({ message: message, to: to, from: from});
```

Comment:
I know this would imply some retesting, but note that sending a property message inside a message object seems a bit strange and redundant to the receiver: `message.data.message`. I think 'content', 'payload' or something else would be more meaningful.

Reply:
also, is not needed to assign "to" to "to", using {to} is enought and less verbose

Reply:
agreed